packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -53,6 +53,12 @@ const propTypes = {
    */
   isReversedChildrenOrder: PropTypes.bool,
 
+  /**
+   * @private
+   * The text for the menu icon. This is read by screenreaders.
+   */
+  iconText: PropTypes.string,
```

Comment:
This looks like the text for the More button based on the code.  Why is the prop private?  Do we not expect it to be set by consumers?

Reply:
We don't expect it to be set by consumers yes, it should only read out "Parent Folders" when I discussed it with @eawww

Reply:
I had it be a prop so that file path can have screen readers read "Parent Folders" while collapsible menu view remains the same and reads "More".

I can change it to menuIconText if that would make it clearer

Reply:
Maybe menuButtonText?  Or is it really just a clickable icon?

Reply:
Just a clickable icon